File: service-java/{{#server.spring}}/src/main/java/{{package_name.value}}/{{#jsonlib.moshi}}/MoshiConfiguration.java
package {{package_name.value}};

import com.squareup.moshi.Moshi;
import org.springframework.context.annotation.*;
import {{package_name.value}}.json.Json;

@Configuration
public class MoshiConfiguration {
	@Bean
	@Primary
	public Moshi getMoshi() {
		var moshiBuilder = new Moshi.Builder();
		Json.setupMoshiAdapters(moshiBuilder);
		return moshiBuilder.build();
	}
}

File: service-kotlin/{{#server.spring}}/src/main/kotlin/{{package.value}}/{{#jsonlib.moshi}}/MoshiConfiguration.kt
package {{package.value}}

import com.squareup.moshi.Moshi
import org.springframework.context.annotation.*
import {{package.value}}.json.setupMoshiAdapters

@Configuration
open class MoshiConfiguration {
    @Bean
    @Primary
    fun getMoshi(): Moshi {
        val moshiBuilder = Moshi.Builder()
        setupMoshiAdapters(moshiBuilder)
        return moshiBuilder.build()
    }
}